package.json

```diff
@@ -29,8 +29,10 @@
     "lerna-preversion": "lerna exec -- npm run lint",
     "lerna-version": "lerna exec -- npm run format && git add -A src",
     "postversion": "git push && git push --tags",
+    "docs-website-build-run": "npm run docs-buildDockerImage && npm run docs-runLocalDocker",
```

Comment:
Added this for laziness: 1 command instead of 2 
:>

Reply:
That's cool, especially for first-time usage.

If you are a returning contributor and you already have the image built you can just run the second command and you'll still see the changes made to the docs source.